doc/admin/troubleshooting.md

```diff
@@ -233,20 +233,62 @@ Refer to the DAOS Environment Variables document for
 more information about the debug system environment.
 
 ## Common DAOS Problems
-
+### Incompatible Agent ####
 When DER_AGENT_INCOMPAT is received, it means that the client library libdaos.so
 is likely mismatched with the DAOS Agent.  The libdaos.so, DAOS Agent and DAOS
 Server must be built from compatible sources so that the GetAttachInfo protocol
 is the same between each component.  Depending on your situation, you will need
 to either update the DAOS Agent or the libdaos.so to the newer version in order
 to maintain compatibility with each other.
 
+### HLC Sync ###
 When DER_HLC_SYNC is received, it means that sender and receiver HLC timestamps
 are off by more than maximum allowed system clock offset (1 second by default).
 
 In order to correct this situation synchronize all server clocks to the same
 reference time, using services like NTP.
 
+### Shared Memory Errors ###
+When DER_SHMEM_PERMS is received it means that this IO Engine lacked the
```

Comment:
I'm sure there are inconsistencies but I think we should try to keep to "I/O Engine" at least when adding new text.

Reply:
Yes, that's a good change.  I will fix that.